src/dfcx_scrapi/tools_ml/utterance_generator_utils.py

```diff
@@ -80,10 +97,19 @@ def clean_string(string_raw):
         )
 
     def _remove_training(self, synthetic_intent_set, existing_phrases: list):
```

Comment:
@Greenford @Hgithubacct add type hinting.
Fix List type hint.

Proper syntax is `List[<type_of_element>]`